modules/core/src/lib/deck-picker.js

```diff
@@ -101,7 +104,12 @@ export default class DeckPicker {
 
   // Private
   _resizeBuffer() {
-    const {gl} = this;
+    const {gl, props} = this;
+
+    if (props._createPickingFBO === false) {
```

Comment:
Can we make this call by checking onHover/onClick are empty and pickable layer count is 0

Reply:
I could certainly add that logic, but I still would like an explicit option to turn off picking buffer creation. 

The problem is that I only need to turn off picking on mobile, not on desktop. And in my app it is a lot of work to override those props in all places, and if it fails (e.g. due to someone maintaining the code and not knowing that all of these things must be aligned) it is hard for me to detect, but accidentally re-enabling the picking buffer can break my app on mobile.

Also, ideally auto-detection logic should come with some test cases etc, so maybe best in a separate PR?

Reply:
Sure. Alternatively we could set a separate, lower device pixel ratio for the picking buffer. That would yield less precise results but leave picking still usable.

Reply:
That would be really neat! 

FWIW, we also tried to use fractional pixel ratios 0.5, 0.3, 0.1 with `useDevicePixels` but not all layers rendered once the ratio got smaller.

Reply:
Can you open an issue for this?

Reply:
> Can you open an issue for this?
- For the auto picking disabling: #4808
- For the device ratio, I'd want to do more testing first before opening a bug. We did not seriously pursue this. In our quick test, a GeoJSON layer seemed to render (looking "smeared out") but our TileLayer did not.

Reply:
Kept rendering at 0.5 but gradually stoped at 0.3, 0.1